Sources/Administrator/InputHandler.cpp

```diff
@@ -101,22 +154,304 @@ void InputHandler::sendProcess(const string& full_command)
   }
 
   string process_name = full_command.substr(full_command.find_first_of(" ")+1);
```

Comment:
i think it works badly, when someone inputs something like: meth_name[lots of whitespaces] path. We should trim all of these